components/TextFields/src/MDCTextInputControllerDefault.m

```diff
@@ -536,7 +550,7 @@ - (CGPoint)placeholderFloatingPosition {
 - (CGFloat)effectiveFloatingScale {
   CGFloat scaleFactor = self.floatingPlaceholderScale
                             ? (CGFloat)self.floatingPlaceholderScale.floatValue
```

Comment:
This spacing/alignment seems a bit odd. Shouldn't it be 4 spaces in from the prior line with CGFloat? Also below.